packages/dashboard-v2/src/components/FileList/FileList.js

```diff
@@ -0,0 +1,74 @@
+import * as React from "react";
+import useSWR from "swr";
+import { useMedia } from "react-use";
+
+import theme from "../../lib/theme";
+
+import { ContainerLoadingIndicator } from "../LoadingIndicator";
+import { Select, SelectOption } from "../Select";
+import { Switch } from "../Switch";
+import { TextInputIcon } from "../TextInputIcon/TextInputIcon";
+import { SearchIcon } from "../Icons";
+
+import FileTable from "./FileTable";
+import useFormattedFilesData from "./useFormattedFilesData";
+
+const FileList = ({ type }) => {
+  const isMediumScreenOrLarger = useMedia(`(min-width: ${theme.screens.md})`);
+  const { data, error } = useSWR(`user/${type}?pageSize=10`);
```

Comment:
I think it should be prefixed with `/` to make sure we're not appending this to some subpage

`/user/${type}?pageSize=10`

also not sure if this worked but all api calls **I think** are prefixed with `/api` so `/api/user/${type}?pageSize=10` (not sure if this comment is not relevant in some other useSWR usages too)

Screen from account.siasky.net network tab:
<img width="381" alt="Screenshot 2022-03-16 at 10 48 19" src="https://user-images.githubusercontent.com/3755029/158562707-1f542244-7bb2-44ac-b4d9-0dd269f67ef9.png">

Reply:
Yeah, the `/api` part is added in the SWR config as a `baseUrl` (line 5), same as prefixing the key with slash (line 8):

https://github.com/SkynetLabs/skynet-webportal/blob/3128a2e7805c1d847e2591b8178fbf2acc0844fc/packages/dashboard-v2/src/lib/swrConfig.js#L4-L8